### PR TITLE
DeviceManager documentation update

### PR DESCRIPTION
**Pull request by kevinuehara**
- Updated device-manager documentation based of last version
  development branch

* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,4 +118,8 @@ work in the future as more strict token checks are implemented in DeviceManager.
 
 ## How to use
 
-There are a few examples on how to use DeviceManager in [this page](./docs/using-device-manager.rst).
+There are a few examples on how to use DeviceManager in [Device Manager Documentation](https://dojotdocs.readthedocs.io/projects/DeviceManager/en/latest/using-device-manager.html).
+
+## API Documentation
+
+URL to api documentation https://dojot.github.io/device-manager/apiary_latest.html for development and latest version of this Device Manager.
```
Comment (mprevide) [inline README.md]: I think the link to development was missing

File: README.md [hunk 2]
```diff
@@ -4,26 +4,70 @@
...
+Device removal          | dojot.device-manager.device                         | Removal message
+Device actuation        | dojot.device-manager.device                         | Actuation message
+Template update         | dojot.device-manager.device                         | Template update message
 
 ## How to run
 
+A docker image is available on dockerhub for pull [here](https://hub.docker.com/r/dojot/device-manager)
+
 If you really need to run DeviceManager as a standalone process (without dojot's wonderful [docker-compose](https://github.com/dojot/docker-compose)), we suggest using the minimal compose
 file available under `docker/compose.yml`. That contains only the set of external systems (postgres
```
Comment (mprevide) [inline README.md]: I think the correct file name is docker-compose.yml

File: README.md [hunk 3]
```diff
@@ -4,26 +4,70 @@
...
+KAFKA_HOST              | kafka host                                          | kafka
+KAFKA_PORT              | kafka port                                          | 9092
+STATUS_TIMEOUT          | kafka timeout                                       | 5
+BROKER                  | kafka topic subject manager                         | http://data-broker
+REDIS_HOST              | redis host                                          | device-manager-redis
+REDIS_PORT              | redis port                                          | 6379
+SUBJECT                 | subject of device-manager                           | dojot.device-manager.device
+DEV_MNGR_CRYPTO_PASS    | password of crypto                                  | none
+DEV_MNGR_CRYPTO_IV      | inicialization vector of crypto                     | none
+DEV_MNGR_CRYPTO_SALT    | salt of crypto                                      | none
```
Comment (mprevide) [inline README.md]: There is also the DEVICE_SUBJECT variable, but it doesn't seem to be used anywhere. 
I believe it is ideal to remove it

File: README.md (hunk 2 above)
Comment (kevinuehara) [reply README.md]: The file actually is called compose.yml, but I missed the directory where the file is (local/compose.yml).
Maybe rename the compose.yml to docker-compose.yml?

File: README.md (hunk 1 above)
Comment (kevinuehara) [reply README.md]: Thanks, I forgot to add the link. I'll add.

File: README.md (hunk 3 above)
Comment (kevinuehara) [reply README.md]: Ok. I'll remove it.

**Pull request merged**